lib/widgets/drive/compare.coffee

```diff
@@ -0,0 +1,40 @@
+_ = require('lodash')
+
+containsDeep = (array, item) ->
+	return _.any(_.map(array, _.partial(_.isEqual, item)))
+
+differenceDeep = (x, y) ->
+	return _.filter(x, _.partial(_.negate(containsDeep), y))
+
+createDiffOperation = (type, element) ->
+	return {
+		type: type
+		drive: element
+	}
+
+###*
+# @summary Detect changes regarding drives between different time intervals
+# @name compare
+# @function
+# @public
+# @memberof visuals.drive
+#
+# @param {Array} - previous drive list
```

Comment:
These params are array of objects. The correct JSDoc syntax for this type is `{Object[]}`.